doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -441,6 +441,27 @@ Here are examples:
    end
    where "n + m" := (plus n m).
 
+Deactivating notations
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. cmd:: Notation Set {? {| @string | @qualid } } {? := @one_term } {? ( {+, @notation_set_flag } ) } {? {| : no scope | : @ident } }
+         Notation Unset {? {| @string | @qualid } } {? := @one_term } {? ( {+, @notation_set_flag } ) } {? {| : no scope | : @ident } }
+
+   .. insertprodn notation_set_flag notation_set_flag
+
+   .. prodn::
+      notation_set_flag ::= in custom @ident
+      | in constr
+      | only parsing
+      | only printing
+
+   Control the activation of notation :token:`string` or
+   :ref:`abbreviation <Abbreviations>` :token:`qualid` or all
+   notations for :token:`one_term`. If given, only notations in scope
+   :token:`ident` are affected (or lonely notations for ``no scope``).
+   The :token:`notation_set_flag` enables to selectively disable or
+   enable parsing or printing.
```

Comment:
```suggestion
   Enables or disables previously defined notations.
   
   :n:`@string`
     The notation string.
     
   :n:`@qualid`
     The qualified name of a previously defined notation or :ref:`abbreviation <Abbreviations>`.
     (explain whether the local/export mechanism applies to notations)

    If given, only notations in scope
   :token:`ident` are affected (or lonely notations for ``no scope``).
   
   :n:`only parsing`
     The notation is enabled or disabled only for parsing.
     
   :n:`only printing`
     The notation is enabled or disabled only for printing.
```

I think this needs a good bit of work to be understandable to users who may be learning or not an expert.  What you have is perhaps reasonable for someone who knows the feature well.

Some details to address:
- First sentence is too dense.
- "control the activation" is vague.  My guess is that the purpose of the command is to dynamically enable and disable previously defined notations.  If correct, please say that.  "Set" and "Unset" suggest creating or removing a notation, perhaps they should be "Enable" and "Disable"
- If correct, say that `string` is just a notation string and `qualid` is the name of a notation **that was previously defined** with `Notation`.  If it could be from `Reserved Notation`, mention that, too.
- I don't understand "all notations for :token:`one_term`" at all.  Probably deserves an example, perhaps in `.. coqtop::` form
- Need to explain `in custom @ident` and `in constr`

I'll take another look after you've revised this.

Reply:
>     * First sentence is too dense.

Indeed, thanks for the suggestion
 
>     * "control the activation" is vague.  My guess is that the purpose of the command is to dynamically enable and disable previously defined notations.  If correct, please say that.  "Set" and "Unset" suggest creating or removing a notation, perhaps they should be "Enable" and "Disable"

Right, changed to `Enable` and `Disable`
 
>     * If correct, say that `string` is just a notation string and `qualid` is the name of a notation **that was previously defined** with `Notation`.  If it could be from `Reserved Notation`, mention that, too.

done
 
>     * I don't understand "all notations for :token:`one_term`" at all.  Probably deserves an example, perhaps in `.. coqtop::` form

Maybe, let me know if you still think this deserves small examples. I could easily add a few.

>     * Need to explain `in custom @ident` and `in constr`

done
 
>     * Should "lonely notations" be listed in the glossary?

Indeed, done